pkg/operator/apiserver/controller/workload/workload.go

```diff
@@ -117,12 +117,12 @@ func (c *Controller) sync() error {
 	}
 
 	if fulfilled, err := c.delegate.PreconditionFulfilled(); !fulfilled || err != nil {
```

Comment:
what are preconditions in practice in (all) our operators?

Reply:
We are changing what a precondition is here. This should go into the go doc in line 44.

Reply:
for OAS-O and CAO preconditions are:

1. wait for the config
2. make sure that mandatory arguments passed to the aggregated APIs are present (i.e. etcd URLs) (this is something I'm going to add)